blueocean/src/test/java/io/jenkins/blueocean/config/JenkinsJSExtensionsTest.java

```diff
@@ -55,7 +55,7 @@ public void test() {
             Assert.assertNotNull(extensionPoints);
 
             if ("blueocean-dashboard".equals(pluginId)) {
-                Assert.assertEquals(7, extensionPoints.size());
+                Assert.assertEquals(9, extensionPoints.size());
```

Comment:
https://issues.jenkins-ci.org/browse/JENKINS-43419 ;) will do it in a another PR to isolate that change.